### PR TITLE
Issue #17 front end

### PR DESCRIPTION
**Pull request by shikher25git**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -166,6 +166,15 @@ <h4 class="label">Settings</h4>
                 <button @click="reScan" class="mj-rescan">
                   Rescan
                 </button>
+              
+              <br>
+              <br>
+              
+                <input placeholder="Username" class="mj-input" type="text">
+                <input type="hidden" v-model="init">
```
Comment (sanket143) [inline index.html:174]: We don't need this input for now. It was a dummy input specifically for `lookupDir`.

File: index.html [hunk 2]
```diff
@@ -166,6 +166,15 @@ <h4 class="label">Settings</h4>
                 <button @click="reScan" class="mj-rescan">
                   Rescan
                 </button>
+              
+              <br>
```
Comment (sanket143) [inline index.html:170]: Using `<br>` for spacing is a bad practice. You can wrap both the field section in divs with the bottom margin as per the need.

**Review by sanket143 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (shikher25git) [reply index.html:174]: If the input isn't there then where will the user enter its username?

Comment (sanket143) [reply index.html:174]: I was talking about the one on line 174 which has `type` = `hidden`

File: index.html (hunk 2 above)
Comment (shikher25git) [reply index.html:170]: Should it be responsive or do I hard code a value?

Comment (sanket143) [reply index.html:170]: You can hard-code it.

Comment (shikher25git) [reply index.html]: Please have a look @sanket143

**Comment by sanket143** [comment]
give @shikher25git 5 points

**Pull request merged**